### PR TITLE
fix: export various classes that are exported in the typings

### PR DESCRIPTION
**Pull request by IRONM00N**
**Please describe the changes this PR makes and why it should be merged:**
The following classes were exported in the typings but not the `index.js` file:
- `ApplicationCommandPermissionsManager`
- `CachedManager`
- `ClientVoiceManager`
- `DataManager`
- `GuildInviteManager`
- `StageInstanceManager`
- `VoiceStateManager`
- `WebSocketManager`
- `WebSocketShard`
- `AnonymousGuild`
- `GuildPreviewEmoji`
- `InviteStageInstance`
- `InviteGuild`
- `PartialGroupDMChannel`
- `StageInstance`
- `StickerPack`
- `Typing`
- `WelcomeChannel`
- `WelcomeScreen`
- `GuildAuditLogsEntry`

**Status and versioning classification:**

- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating

### CODE REVIEW

**Review by kyranet — APPROVED** [review]

File: typings/index.d.ts [hunk 1]
```diff
@@ -630,7 +630,7 @@ export class GuildAuditLogs {
   public toJSON(): unknown;
 }
 
-export class GuildAuditLogsEntry {
+declare class GuildAuditLogsEntry {
```
Comment (vladfrangu) [inline typings/index.d.ts:633]: While I agree that the class isn't *exported* by us directly (maybe it should be..), it should still be available to TS users. Thoughts @discordjs/the-big-3?

**Review by vladfrangu — DISMISSED** [review]

File: typings/index.d.ts (hunk 1 above)
Comment (SpaceEEC) [reply typings/index.d.ts:633]: I think we should just export it.

Comment (IRONM00N) [reply typings/index.d.ts:633]: @SpaceEEC Just in the typings or actually exported?

Comment (SpaceEEC) [reply typings/index.d.ts:633]: I'd say actually exported.

**Comment by IRONM00N** [comment]
`GuildAuditLogsEntry` is now exported.

**Review by iCrawl — APPROVED** [review]

**Review by vladfrangu — APPROVED** [review]

**Review by SpaceEEC — APPROVED** [review]

**Pull request merged**